pkg/driver/camera/camera_linux.go

```diff
@@ -70,6 +72,7 @@ type camera struct {
 	started         bool
 	mutex           sync.Mutex
 	cancel          func()
+	prevFrameTime   time.Time
```

Comment:
I like that idea! I'm not sure how it'd be best implemented but I have a few ideas:

1. We could make this struct public so users can cast their `Driver` interface into this concrete type and set `DiscardFramesOlderThan`.
2. We could add a generic method for setting properties to the `Driver` interface. Something like `Set(prop map[string]interface{})`
3. Add a non-const global variable.

I think 2 would make the most sense `Driver` isn't camera specific. All would require locking the mutex which isn't exposed, so 1 and 3 seems dangerous unless you expose a mutex.

Reply:
prop.Video would work as the input to this function.

Reply:
I don't think it needs to be a constraint since that's a selection criteria.

Reply:
`prop.Video` and `prop.VideoConstraints` are merged in `prop.go::merge` using reflections. This is only possible because the two structs are identical. If they differ we'll get a `reflect: Field index out of range` error unless we handle this field specially in that function.

Reply:
oh lame, okay!